src/julia-parser.scm

```diff
@@ -78,7 +78,7 @@
 ; operators that are special forms, not function names
 (define syntactic-operators
   (append! (add-dots '(= += -= *= /= //= |\\=| ^= ÷= %= <<= >>= >>>= |\|=| &=))
-           '(:= --> $= => && |\|\|| |.| ... ->)))
+           '(:= --> $= ⊻= => && |\|\|| |.| ... ->)))
```

Comment:
@tkelman I suppose this `⊻=` should then also move one line up to `add-dots`, right?